src/scripts/form-validation.js

```diff
@@ -0,0 +1,105 @@
+const emailEl = document.querySelector('#email');
+const userName = document.querySelector('#Name');
+const userLastName = document.querySelector('#Last_Name');
+const message = document.querySelector('#message');
+const form = document.querySelector('#Form-form');
+
+const isEmailValid = (email) => {
+  const re = /[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/;
+  return re.test(email);
+};
```

Comment:
[REQUIRED] Nice implementation of validating the email input format :+1: However, considering the requirements of this milestone, I have to ask you to add logic to this function so it can also check if the email content is presented in all lower case characters, otherwise it should present the user respective error message and prevent the form from submission. When I tried to submit the form with the email `Test@mail.com`, it actually did accept the value, which should not be the case as per validation requirements.